### PR TITLE
updated IDAHelper class

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

**Comment by hellwolf** [comment]
@pi0neerpat  please review the breaking changes in IDAHelper and its updated documentation!

File: packages/js-sdk/test/InstantDistributionAgreementV1Helper.test.js [hunk 1]
```diff
@@ -312,23 +312,23 @@ contract("InstantDistributionAgreementV1Helper helper class", (accounts) => {
...
                 units: units.toString(),
-                sender: publisher,
             });
 
             await sf.ida.approveSubscription({
                 superToken: superToken.address,
                 indexId,
                 publisher,
-                sender: subscriber,
+                subscriber: subscriber,
```
Comment (pi0neerpat) [inline packages/js-sdk/test/InstantDistributionAgreementV1Helper.test.js:331]: if the sender is always the subscriber, then this is ok

Comment (hellwolf) [reply packages/js-sdk/test/InstantDistributionAgreementV1Helper.test.js:331]: yes it is.

The purpose of these changes is to keep all the parameters consistent. Only some operation would require "sender" parameter to tell from which account the transaction should be (including claim and deleteSubscription)

**Comment by hellwolf** [comment]
@pi0neerpat ready

**Review by pi0neerpat — APPROVED** [review]
LGTM I can update docs this week

**Pull request merged**